Makefile

```diff
@@ -27,6 +27,8 @@ test-slow-chop: og
 	turnt --env chop_test test/*.gfa
 
 test-slow-crush: og
+	-turnt --save --env crush_oracle test/*.og
+	-turnt --env crush_test test/*.gfa
 	-turnt --save --env crush_oracle test/handmade/crush*.gfa
```

Comment:
Just wanted to provide a pointer to the thing I'm talking about in the comment here: https://github.com/cucapra/pollen/pull/50#issuecomment-1507803443